client/request/commit.go

```diff
@@ -60,3 +62,20 @@ func (c CommitSelect) ToSelect() *Select {
 		ChildSelect: c.ChildSelect,
 	}
 }
+
+// Implements the subscriptionSelector interface in internal/db/subscriptions.go
+// We can safely ignore the first parameter (docID) for now
+// since its always copied from the original subscription request
+func (c CommitSelect) ToSubscriptionSelect(_, cid string) Selection {
```

Comment:
nitpick: I want to enable a linter that catches typos and wrong (non-matching) function names as first word somewhere in the nesr future, so in that spirit leaving this nitpick

```suggestion
// ToSubscriptionSelect implements the subscriptionSelector interface in internal/db/subscriptions.go
// We can safely ignore the first parameter (docID) for now
// since its always copied from the original subscription request
func (c CommitSelect) ToSubscriptionSelect(_, cid string) Selection {
```